python/hidet/graph/ops/softmax.py

```diff
@@ -16,6 +16,9 @@
 from hidet.ir.builders import StmtBuilder
 from hidet.ir.primitives import active_mask, shfl_down_sync, shfl_sync
 from .utils import Task, TensorNode, compute, reduce
+from typing import List, Union
```

Comment:
Move to the top.

Remember to run format & lint, see https://docs.hidet.org/stable/developer-guides/contributing.html#contributing